packages/aws-fargate/src/activate.js

```diff
@@ -18,10 +18,13 @@ const { normalizeConfig } = coreUtil;
 let logger = consoleLogger;
 
 const config = normalizeConfig({});
+config.logger = logger;
 
 function init() {
-  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL) {
-    logger.setLevel(process.env.INSTANA_DEBUG ? 'debug' : process.env.INSTANA_LOG_LEVEL);
+  const logLevel = process.env.INSTANA_DEBUG === 'true' ? 'debug' : process.env.INSTANA_LOG_LEVEL;
```

Comment:
In my understanding, we don't have to set the INSTANA_DEBUG to 'true' or anything. We can just set the value as say `test` || `1` or any "TRUTHY" values for the debugger to be enabled.

Find the [public doc here](https://www.ibm.com/docs/en/instana-observability/current?topic=nodejs-collector-configuration#log-level-configuration)

> You can also configure the log level by setting the environment variable INSTANA_LOG_LEVEL to either debug, info, warn or error. Finally, setting INSTANA_DEBUG to any non-empty string will set the log level to debug.
> 

This applies to all the similar changes.